testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -286,6 +286,10 @@
   $server.run("zypper --non-interactive remove #{package}-formula")
 end
 
+When(/^I synchronize the Salt execution modules$/) do
+  $server.run("salt '*' saltutil.sync_modules")
```

Comment:
Is it necessary to target all minions by "*"? Shouldn't we target only the needed one? Apart of that LGTM :-)

Reply:
Minion? Aren't we acting through the server and a formula applied on it?

Reply:
yep, but this is really minor (we have only a few systems to target)

Reply:
> Is it necessary to target all minions by "*"? Shouldn't we target only the needed one?

@lkotek good remark - maybe, yes, I'll have a look. It might also be that it's "safer" to target them all.

> Minion? Aren't we acting through the server and a formula applied on it?

Yes. The formulas don't work if you don't synchronize the "Salt execution modules" on the minions after you instal them. Don't ask me why.

Reply:
> Is it necessary to target all minions by "*"? Shouldn't we target only the needed one?

Done, please check new version.